packages/plugin-ext/src/main/node/paths/plugin-paths-service.ts

```diff
@@ -102,6 +104,22 @@ export class PluginPathsServiceImpl implements PluginPathsService {
         }
     }
 
+    private async getLogsDirFsPath(): Promise<string> {
+        return new Promise<string>(resolve => {
+            this.envServer.getConfigDirUri()
+                .then(configDirUri => FileUri.fsPath(configDirUri))
+                .then(configDirFsPath => resolve(path.join(configDirFsPath, PluginPaths.PLUGINS_LOGS_DIR)));
+        });
+    }
+
+    private async getWorkspaceStorageDirFsPath(): Promise<string> {
+        return new Promise<string>(resolve => {
```

Comment:
the same as above

Reply:
Let's do, I do not have any strong preference on this.